metrics/metrics.go

```diff
@@ -78,72 +69,38 @@ var (
 		TagKeys:     []tag.Key{Direction, UseFD, Op},
 	}
 
-	RcmgrAllowConnView = &view.View{
-		Measure:     RcmgrAllowConn,
+	RcmgrPeerView = &view.View{
+		Measure:     RcmgrPeer,
 		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{Direction, UseFD},
+		TagKeys:     []tag.Key{PeerID, Op},
 	}
-	RcmgrBlockConnView = &view.View{
-		Measure:     RcmgrBlockConn,
-		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{Direction, UseFD},
-	}
-	RcmgrAllowStreamView = &view.View{
-		Measure:     RcmgrAllowStream,
-		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{PeerID, Direction},
-	}
-	RcmgrBlockStreamView = &view.View{
-		Measure:     RcmgrBlockStream,
-		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{PeerID, Direction},
-	}
-	RcmgrAllowPeerView = &view.View{
-		Measure:     RcmgrAllowPeer,
-		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{PeerID},
-	}
-	RcmgrBlockPeerView = &view.View{
-		Measure:     RcmgrBlockPeer,
-		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{PeerID},
-	}
-	RcmgrAllowProtoView = &view.View{
-		Measure:     RcmgrAllowProto,
-		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{ProtocolID},
-	}
-	RcmgrBlockProtoView = &view.View{
-		Measure:     RcmgrBlockProto,
-		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{ProtocolID},
-	}
-	RcmgrBlockProtoPeerView = &view.View{
-		Measure:     RcmgrBlockProtoPeer,
-		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{ProtocolID, PeerID},
-	}
-	RcmgrAllowSvcView = &view.View{
-		Measure:     RcmgrAllowSvc,
+
+	RcmgrProtoView = &view.View{
+		Measure:     RcmgrProto,
 		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{ServiceID},
+		TagKeys:     []tag.Key{ProtocolID, Op},
 	}
-	RcmgrBlockSvcView = &view.View{
-		Measure:     RcmgrBlockSvc,
+
+	RcmgrProtoPeerView = &view.View{
+		Measure:     RcmgrProtoPeer,
 		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{ServiceID},
+		TagKeys:     []tag.Key{ProtocolID, PeerID, Op},
 	}
-	RcmgrBlockSvcPeerView = &view.View{
-		Measure:     RcmgrBlockSvcPeer,
+
+	RcmgrSvcView = &view.View{
+		Measure:     RcmgrSvc,
 		Aggregation: view.Count(),
-		TagKeys:     []tag.Key{ServiceID, PeerID},
+		TagKeys:     []tag.Key{ServiceID, Op},
 	}
-	RcmgrAllowMemView = &view.View{
-		Measure:     RcmgrAllowMem,
+
+	RcmgrSvcPeerView = &view.View{
+		Measure:     RcmgrSvcPeer,
 		Aggregation: view.Count(),
+		TagKeys:     []tag.Key{ServiceID, PeerID, Op},
 	}
```

Comment:
```suggestion
```
Monitoring of peer_id stuff needs to be done by logging not prometheus metrics.

Reply:
resolved on the latest commit.